log/aix_utmp.ksy

```diff
@@ -0,0 +1,75 @@
+meta:
+  id: aix_utmp
+  title: utmp log file, IBM AIX version
+  license: CC0-1.0
+  endian: be
+doc: |
+  This spec can be used to parse utmp, wtmp and other similar as created by
+  IBM AIX. Spec is based on IBM Knowledge Base article:
+  https://www.ibm.com/support/knowledgecenter/en/ssw_aix_71/com.ibm.aix.files/utmp.h.htm
+seq:
+  - id: records
+    type: record
+    repeat: eos
+types:
+  record:
+    seq:
+    - id: ut_user
+      doc: User login name
+      size: 256
+      type: str
+      encoding: utf-8
+    - id: ut_id
+      doc: /etc/inittab id
+      size: 14
+      type: str
+      encoding: utf-8
+    - id: ut_line
+      doc: device name (console, lnxx)
+      size: 64
+      type: str
+      encoding: utf-8
```

Comment:
Most likely ASCII? I'm not sure if there's any possibility of UTF-8 characters appearing here.